### PR TITLE
add extra jira fields to jql

### PR DESCRIPTION
**Pull request by abloch**

### CODE REVIEW

File: jira/actions/jql_actions.py [hunk 1]
```diff
@@ -3,13 +3,21 @@
 from .issue_actions import get_jira_url
 from .. import action_store as action_store
 
+JIRA_FIELDS = [
+    "description", "assignee", "created", "summary", "status", "issuetype",
+    "customfield_10021", "updated", "subtasks", "customfield_10021", "customfield_10020",
```
Comment (tal-mogendorff) [inline jira/actions/jql_actions.py:8]: There is Duplicate "customfield_10021" is it should be ? 
@shakedaskayo / @abloch

**Review by tal-mogendorff — COMMENTED** [review]
Test

File: jira/actions/jql_actions.py (hunk 1 above)
Comment (abloch) [reply jira/actions/jql_actions.py:8]: now I can see it

**Review by tal-mogendorff — APPROVED** [review]
Seems good

**Pull request merged**